endpoints.php

```diff
@@ -285,6 +285,13 @@
             'callback' => 'getTapestryContributors',
         ],
     ],
+    'GET_ALL_ROLES' => (object) [
+        'ROUTE' => '/all_roles',
```

Comment:
Small nitpick, but I think a better endpoint for this would be `/roles`. I think it better conforms to the standard of REST being organized by resources.